src/providers/ipa/ipa_views.c

```diff
@@ -565,6 +574,31 @@ static void ipa_get_trusted_override_done(struct tevent_req *subreq)
     if (reply_count == 0) {
         DEBUG(SSSDBG_TRACE_ALL, "No override found with filter [%s].\n",
                                 state->filter);
+
+        /* If looking up the auto private group for a user with a login name override,
+         * switch from BE_REQ_GROUP to BE_REQ_USER to fetch the user.
```

Comment:
Imo this comment is not very clear. I would suggest something like "If there is no group override found during a group lookup by name in a MPG domain switch from BE_REQ_GROUP to BE_REQ_USER to check if there might be a user override found with the given name and the requested group is a user private group."

Reply:
Thanks.